qemu-test-init

```diff
@@ -119,15 +109,29 @@ if type casync; then
   export RAUC_TEST_CASYNC=1
 fi
 
-# rauc system config & service start
-if [ -n "$SERVICE" ]; then
-  mkdir /tmp/rauc
-  cp qemu-test-rauc-config /tmp/rauc/system.conf
-  cp test/openssl-ca/dev-ca.pem /tmp/rauc/ca.cert.pem
-  if [ -n "$SERVICE_BACKTRACE" ] && type gdb; then
-    gdb --batch --ex "run" --ex "thread apply all bt" --args rauc service --conf=/tmp/rauc/system.conf &
-  else
-    rauc service --conf=/tmp/rauc/system.conf &
+# rauc system config
+rm -rf /etc/rauc
+mkdir /etc/rauc /tmp/rauc
```

Comment:
Do we still need to create `/tmp/rauc`?

Reply:
I didn't want to risk changing the behavior in this PR.